pkg/utils/ptr/ptr.go

```diff
@@ -18,6 +18,18 @@ import (
 	v1 "k8s.io/api/core/v1"
 )
 
+func String(s string) *string {
+	return &s
+}
+
+func StringOrEmpty(s *string) string {
```

Comment:
Check out [Knative's ptr.StringValue()](https://github.com/knative/pkg/blob/main/ptr/value.go#L68). I implemented all of these in Knative PKG to we should be able to deprecate all of these functions except for the pod specific ones.